share/testing.go

```diff
@@ -26,30 +26,30 @@ import (
 	"github.com/celestiaorg/celestia-node/ipld"
 )
 
-// RandLightServiceWithSquare provides a share.Service filled with 'n' NMT
+// randLightServiceWithSquare provides a share.Service filled with 'n' NMT
 // trees of 'n' random shares, essentially storing a whole square.
-func RandLightServiceWithSquare(t *testing.T, n int) (*Service, *Root) {
+func randLightServiceWithSquare(t *testing.T, n int) (*Service, *Root) {
```

Comment:
What's the reasoning behind privatizing all these? It's normal to provide testing utilities to outsiders. Thought we could make em public once the use case comes, and hide the for now. Not strictly opposed to this

Reply:
They were just privatized because `Service` was `service` for a while, and it was a better solution to unexport than to tag each method with a nolint revive. But We can also keep them unexported if its not a big deal? I like the thinking of "we could make them public once the use case comes"

Reply:
Yeah, let's keep it unexported. It is fine